content/vault/v1.21.x (rc)/content/partials/plugins/tune-keymgmt-steps.mdx

```diff
@@ -0,0 +1,25 @@
+<Note>
+
+Vault supports configuration updates (tune) on mounted plugins for the
+following:
+
+- `keymgmt`
+
+</Note>
```

Comment:
We should make this a separate partial so we can add to the list as new plugins are added.